pkg/storaged/pages.jsx

```diff
@@ -816,14 +816,14 @@
     return (
         <Page id="storage">
             { parent_crumbs.length > 0 &&
-            <PageBreadcrumb stickyOnBreakpoint={{ default: "top" }}>
+            <PageBreadcrumb {...(!client.in_anaconda_mode() ? { stickyOnBreakpoint: { default: "top" } } : {})}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19876-20240123-135148-fddbc75a-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
What does stickyOnBreakpoint actually do? It is not documented for PF 5.1.0, as far as I can see. Can we just remove it unconditionally? @garrett?

Reply:
Ah, sorry, this is `PageBreadcrumb` not `Breadcrumb`, which documents it. But I am still not sure what it actually does...

Reply:
(I was confused by what this does, but it seems to be necessary to control some padding below the breadcrumb.)

Can we write this as `stickyOnBreakpoint={ client.in_anaconda_mode() ? {} : {` default: false }}` ? That's less magic.

Reply:
It makes that section sticky which has as a result visual effects like shadow bellow the bottom.